src/login/logind-dbus.c

```diff
@@ -1914,7 +1922,23 @@ static int method_poweroff(sd_bus_message *message, void *userdata, sd_bus_error
                         "org.freedesktop.login1.power-off-multiple-sessions",
                         "org.freedesktop.login1.power-off-ignore-inhibit",
                         NULL,
-                        error);
+                        error,
+                        false);
+}
+
+static int method_poweroff_ext(sd_bus_message *message, void *userdata, sd_bus_error *error) {
```

Comment:
Instead of adding all these new helpers that are nearly identical, you can use the same and then check the method name:

Instead of adding all these new helpers, you can use the existing ones and check the method name with something like

if (sd_bus_message_is_method_call(message, NULL, "PowerOffWithInhibitors")